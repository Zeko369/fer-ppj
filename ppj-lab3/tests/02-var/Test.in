<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 3
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 y
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        IDN 2 x
       <T_lista>
        $
      <E_lista>
       OP_PLUS 2 +
       <E>
        <T>
         <P>
          BROJ 2 3
         <T_lista>
          $
        <E_lista>
         $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 z
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         BROJ 3 5
        <T_lista>
         $
       <E_lista>
        OP_PLUS 3 +
        <E>
         <T>
          <P>
           BROJ 3 8
          <T_lista>
           $
         <E_lista>
          $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 4 a
       OP_PRIDRUZI 4 =
       <E>
        <T>
         <P>
          IDN 4 x
         <T_lista>
          $
        <E_lista>
         OP_PLUS 4 +
         <E>
          <T>
           <P>
            IDN 4 y
           <T_lista>
            $
          <E_lista>
           OP_PLUS 4 +
           <E>
            <T>
             <P>
              IDN 4 z
             <T_lista>
              $
            <E_lista>
             $
     <lista_naredbi>
      $
