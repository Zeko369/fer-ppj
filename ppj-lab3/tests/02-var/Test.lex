IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 3
IDN 2 y
OP_PRIDRUZI 2 =
IDN 2 x
OP_PLUS 2 +
BROJ 2 3
IDN 3 z
OP_PRIDRUZI 3 =
BROJ 3 5
OP_PLUS 3 +
BROJ 3 8
IDN 4 a
OP_PRIDRUZI 4 =
IDN 4 x
OP_PLUS 4 +
IDN 4 y
OP_PLUS 4 +
IDN 4 z
