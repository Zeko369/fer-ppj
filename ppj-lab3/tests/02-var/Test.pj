x = 3
y = x + 3
z = 5 + 8
a = x + y + z
