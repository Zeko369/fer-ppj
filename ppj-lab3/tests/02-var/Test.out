2 1 x
4 1 x
4 2 y
4 3 z
